protocol/flows/src/v5/blockrelay/flow.rs

```diff
@@ -116,7 +116,7 @@ impl HandleRelayInvsFlow {
                 }
             }
 
-            if self.ctx.is_ibd_running() && !session.async_is_nearly_synced().await {
+            if self.ctx.is_ibd_running() && !self.ctx.is_nearly_synced(&session).await {
```

Comment:
Not sure about this one. If the node is on IBD but *yet* sync rate is below threshold and thus mining, I think relay blocks should be processed as well.